src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -162,187 +191,161 @@ When working to improve the user experience, it's best to think of the process a
+- [Optimizing LCP](/optimize-lcp/)
+- [Optimizing CLS](/optimize-cls/)
+- [Optimizing FID](/optimize-fid/)
+- [Optimizing INP](/optimize-inp/)
 
 #### Uncover opportunities with Lighthouse
 
-Though PageSpeed Insights runs Lighthouse for you, it's more convenient during local development to run Lighthouse from Chrome's DevTools.
+PageSpeed Insights runs Lighthouse for you, but for local development it is also possible to run Lighthouse from Chrome DevTools which is useful to validate fixes locally.
 
 <figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/k6HffrY6tUbreyzi2mVk.png", alt="A screenshot of a Lighthouse report within Chrome's DevTools. The report breaks down scores across five categories, with the report focused on the 'Performance' category, with results at the bottom of the report window.", width="800", height="526" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/k6HffrY6tUbreyzi2mVk.png", alt="A screenshot of a Lighthouse report within Chrome DevTools. The report breaks down scores across five categories, with the report focused on the 'Performance' category, with results at the bottom of the report window.", width="800", height="526" %}
 </figure>
 
-{% Aside %}
-Out of the box, Lighthouse only assesses the user experience during page startup. Since it's a lab tool, it also excludes FID in favor of TBT and [Time to Interactive (TTI)](/tti/). If you need to profile specific interactions or page behaviors that occur after startup, read the [Lighthouse user flows guide](/lighthouse-user-flows/) for more information.
-{% endAside %}
-
-Lighthouse runs simulate a mid-tier mobile device on a throttled slow 4G connection. This may find issues that wouldn't ordinarily appear on high-speed devices or fast internet connections.
+A key point is to validate that the Lighthouse audit replicates the issues you are trying to solve (for example, slow LCP, or CLS issues). Out of the box, Lighthouse only assesses the user experience during page load. Since it's a lab tool, it also excludes FID and INP in favor of TBT.
 
-Even so, this simulated throttling may not be representative of the variety of user experiences among your website's user base at the 75th percentile. However, these metrics are an indicator of where performance problems exist, and may translate into better performance overall in the field if the problems Lighthouse finds are addressed.
+{% Aside 'caution' %}
+By default, Lighthouse runs simulating a mid-tier mobile device on a throttled slow 4G connection. This may find issues that wouldn't ordinarily appear on high-speed devices or fast internet connections. This simulated throttling may not be representative of the variety of user experiences among your website's user base at the 75th percentile. However, these metrics are an indicator of where performance problems exist, and may translate into better performance overall in the field if the problems Lighthouse finds are addressed.
```

Comment:
```suggestion
By default, Lighthouse simulates a mid-tier mobile device on a throttled slow 4G connection. This may find issues that wouldn't ordinarily appear on high-speed devices and fast internet connections. This simulated throttling may not be representative of the variety of user experiences among your website's user base at the 75th percentile. However, these metrics are an indicator of where performance problems exist, and may translate into better performance overall in the field if the problems Lighthouse finds are addressed.
```